.github/workflows/verify-version.yml

```
@@ -0,0 +1,35 @@
+name: Verify plugin version
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+
+jobs:
+  verify-version:
+    name: Verify that plugin version matches SQLITE_DRIVER_VERSION constant
```

What do you think about this slightly longer name?

```suggestion
    name: Assert the WordPress plugin header declares the same version as the SQLITE_DRIVER_VERSION constant
```

---

Thank you!